### PR TITLE
Contribute ribbon disappears on screen reduction

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_heart: Ready for thumbs up.  Easy one - Contribute ribbon disappears on screen reduction.

### CODE REVIEW

**Comment by ericanaughton** [comment]
when i decreased the screen size, i saw this..
image

**Comment by ericanaughton** [comment]
^ relating to the above comment: this hides in the top corner, so you only see it when you scroll up. but would be good to have it disappear completely. or until you find the solution for creating a new ribbon shape.

**Comment by ericanaughton** [comment]
good work @kathyoleary :+1:

File: css/style.css [hunk 1]
```diff
@@ -255,6 +255,14 @@ h4 {
   box-shadow: 0px 0px 8px rgba(0, 0, 0, 0.3);
 }
 
+@media all and (max-width: 1140px) {
+  .bookmark {
+    visibility: hidden;
+    height: 0;
+  }
+}
```
Comment (ericanaughton) [inline css/style.css:316]: you might need a new line after 263, i think?

File: css/style.css [hunk 2]
```diff
@@ -303,14 +311,15 @@ h4 {
...
+.bookmark img {
   width: 40px;
   margin-left: 20px;
  }
 
-.bookmark:hover{
+.bookmark:hover {
   -webkit-filter: drop-shadow(0 2px 6px hsla(0, 0%, 0%, 0.75));
 }
+}
```
Comment (ericanaughton) [inline css/style.css]: looks like an extra `}` on 322

Comment (ericanaughton) [inline css/style.css]: found this too.. need a space before `{`

image

image

**Comment by ericanaughton** [comment]
and overall, the css file could use some abc sorting... i suppose that can be a different issue.

**Comment by kathyoleary** [comment]
Added the spaces, but the } on 322 closes the media query started on 265.

**Comment by kathyoleary** [comment]
do we want the CSS in abc order or html order corresponding to where its at on the page?

**Comment by ericanaughton** [comment]
the styles within the classes should be abc order... you can install a package in your text editor to do it for you.. if you have atom its called 'sort'

**Comment by ericanaughton** [comment]
if the `}` on 322 is needed, then there needs to be proper indentation

**Comment by kathyoleary** [comment]
ok..question then because I misunderstood this...

In https://github.com/LMHT/LMHT.github.io/issues/96, it said to:

In CSS, all styles are alphabetized in each class
In CSS, all classes are in order, relating to where they are in HTML

So what is in HTML order?

**Comment by kathyoleary** [comment]
@ericanaughton Can you double check this? You have the best eye for this and let me know if it gets a thumb.  Thanks!

**Comment by rylnd** [comment]
I loaded it up, looks good to me! :+1:

**Comment by missingdink** [comment]
:+1:

**Pull request merged**